libs/components/forms/src/lib/modules/file-attachment/file-drop.component.html

```diff
@@ -89,23 +89,30 @@
             }}
           </div>
         </div>
-        <div class="sky-margin-stacked-sm">
+        <div class="sky-margin-stacked-xs">
           <input
             type="text"
             class="sky-form-control"
             [attr.aria-label]="
               linkUploadAriaLabel ||
               ('skyux_file_attachment_file_upload_link_label' | skyLibResources)
             "
-            [attr.placeholder]="
-              'skyux_file_attachment_file_upload_link_placeholder'
-                | skyLibResources
-            "
+            [attr.aria-describedby]="hintText.id"
```

Comment:
I'd add a unit test confirming this change.

Reply:
Added